Creating the headline

tomasesquivelgc opened the pull request:
## In this branch I have:
  - Created a headline
  - Added the text with it's corresponding style
  - added the social media bar and applied responsiveness to position
  - inserted social media images and background images

### As a side note, it has come to my attention that perhaps I did not need to create a desktop version as well. By this pull request I submit my work so far and wait for answers to the questions: 
  - Did I go to far?
  - Is that why it took me longer than expected?
  - What should I do now?

tomasesquivelgc committed:
created headline

copied text and style added

tomasesquivelgc committed:
added social media tab

social-media links

tomasesquivelgc committed:
fixed images linking and media responsiveness

now links will move to the left in a bigger screen

tomasesquivelgc committed:
style.css small change in syntax

tomasesquivelgc committed:
added background image

tomasesquivelgc committed:
moved background element to bottom right corner

tomasesquivelgc committed:
fixed typying

tomasesquivelgc committed:
more typing fix

iambenkis commented on index.html:
```diff
@@ -28,5 +28,18 @@
+            <a href="#"><img src="./Normal_Button/Icon-Linkedin.png"></a>
+            <a href="#"><img src="./Normal_Button/angelist-1.png"></a>
+            <a href="#"><img src="./Normal_Button/Icon-Twitter.png"></a>
+            <a href="#"><img src="./Normal_Button/Icon-Medium.png"></a>
```
- [x] Good job setting these social media links to your first page of your portfolio 👍 , However I think it could be better to use a `ul` and `li` tags. Kindly improve that for semantic reason 😄

iambenkis commented on index.html:
```diff
@@ -28,5 +28,18 @@
+    <div class="headline">
+        
+        <h1>Hey There.<br>I'm Tomás</h1>
+        <h2>I'm a web developer</h2>
```
- [x] Instead of using a `div` tag it could be better if you use a `section` tag to meet the requirements. Also using a `section` tag will make your page more semantic 😸

iambenkis reviewed (changes requested):
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @tomasesquivelgc 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [x] Kindly use an `a` tag to the logo instead of the `p` tag. Remember that for a good user experience you should use an `a` tag to logo 👍 

https://github.com/tomasesquivelgc/Portfolio_Mobile_First/blob/146f4f58ca18fe6a478ecb19f1ae9b74036ad810/index.html#L14-L16

- [x] Kindly remove the broken link of the logo found in your README file, 


- [x] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tomasesquivelgc committed:
changes following code reviewer

  - logo element was changed from <p> to <a>
  - broken link from README removed
  - social media links now reside in a <ul> element each with a <li> tag
  - headline element changed from <div> to <section>

tomasesquivelgc committed:
fixed typing

according to linters

tomasesquivelgc committed:
fixed indentation in .css

tomasesquivelgc commented:
Changes made according to reviewer

huseyinbiyik commented on index.html:
```diff
@@ -12,12 +12,12 @@
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@600&family=Roboto:wght@700&display=swap" rel="stylesheet">
 <body>
     <nav class="toolbar" id="toolbar">
```
[OPTIONAL] Since @iambenkis didn't notice, I will ask this as optional. Kindly use a `<header>` wrapper to wrap your nav and logo. In this way, you can make your page more semantic and accessible.
Typically the header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
So, that order would be good for you.
```html
<header>
  <a href="#"> My logo...
  <nav>
    <ul><li><a...
   </nav> 
 </header>

```

huseyinbiyik commented on index.html:
```diff
@@ -12,12 +12,12 @@
 <body>
     <nav class="toolbar" id="toolbar">
-        <p class="logo">My Logo</p>
+        <a class="logo">My Logo</a>
```
[OPTIONAL] You changed your logo from p to an element but let's add a href value so it can get clickable. Check this l[ink](https://www.w3schools.com/tags/att_a_href.asp) to learn more about a tag.

huseyinbiyik commented on index.html:
```diff
@@ -12,12 +12,12 @@
                 </span></button>
-            <div class="dropdown-menu">
+            <ul class="dropdown-menu">
                 <button href="#">Portfolio</button>
```
[OPTIONAL] I think it would be better if you use `<a>` tag for navigation. Button tag is mostly used for opening popup, deleting a task from the to-do list etc. If you are leading the user to a URL, it would be better to use `<a>`tag with its href value 👍

huseyinbiyik commented on style.css:
```diff
@@ -61,10 +66,56 @@ body {
+  cursor: pointer;
+}
+
 @media (min-width: 375px) {
```
[OPTIONAL] Since we are following the mobile-first approach, there is no need to use mobile media query. Otherwise, your code will not DRY and you will have to repeat CSS values for mobile and desktop versions.

huseyinbiyik commented on README.md:
```diff
@@ -27,7 +27,6 @@ After you're finished please remove all the comments and instructions!
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
 
   <h3><b>Microverse README Template</b></h3>
```
[OPTIONAL] Unfortunately the previous reviewer should have invalidate your Pull Request since you have an touched README Template. You need to customize your README file with your custom content. You used [Microverse Readme template](https://github.com/microverseinc/readme-template) but make sure all the main sections present in yours and placeholders are replaced with your project and personal data. Please remove all the instructions targeting you. For example: "Describe how a new developer could make use of your project." The only optional section is the FAQ. Please remove the template logo on the top, if you have your own logo you can replace it with this one, otherwise please delete it. This is crucial to have a professional and instructive repository. 👌 
![Screen Recording 2023-03-24 at 10 18 10 PM](https://user-images.githubusercontent.com/25887253/227621296-953f5feb-16ca-4210-bc45-6e417d6db8bd.gif)

huseyinbiyik reviewed (approved):
Hi @tomasesquivelgc 

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![Alt Text](https://media.giphy.com/media/wZjlCH43M3M0U/giphy.gif)



## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- PR description is descriptive 📝

## Optional suggestions
-  Please make sure your commit messages are meaningful and descriptive for your next commits. Use imperative tense and capitalize the first word. For example, 
👌"Add logo to navigation bar"
👌"Fix close button functionality on mobile navbar"
❌ "Added logo to navigation bar"
❌ "Added HTML" 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

huseyinbiyik commented on index.html:
```diff
@@ -28,5 +28,19 @@
         </div>
     </nav>
 
+    <section class="headline">
```
[OPTIONAL] Kindly make sure your design matches with the Figma template. Currently, there are some problems with nav and headline. Please try to give a display flex instead of a fixed value. 
<img width="635" alt="Screenshot 2023-03-24 at 10 09 44 PM" src="https://user-images.githubusercontent.com/25887253/227623340-7904ac37-ebdd-4125-9ce8-1c77fea2fabb.png">

<img width="452" alt="Screenshot 2023-03-24 at 10 37 55 PM" src="https://user-images.githubusercontent.com/25887253/227623326-4cea5d2a-570b-4441-8898-4ae874228be7.png">

tomasesquivelgc replied:
Hello, thanks for your review. In my browser this problem does not happen. Any idea why that could be? the toolbar width is set to 100% so there shouldn't be any empty space.

![image](https://user-images.githubusercontent.com/117373513/227629514-ab7ccd06-3473-4f17-b101-749bfb6a35e7.png)

huseyinbiyik replied:
Hey @tomasesquivelgc. I think It needs to be a 100%.
<img width="698" alt="Screenshot 2023-03-25 at 12 17 53 AM" src="https://user-images.githubusercontent.com/25887253/227643697-b1156877-3bd4-41fb-8af4-d4abc1e2618c.png">

tomasesquivelgc committed:
changes according to reviewer

 - LICENCE add
 - <header> element wraping navbar
 - navigation buttons changed to <a> elements
 - add href to logo
 -

tomasesquivelgc committed:
syntax correct
